### PR TITLE
Add RealtimeChangeHandler arg validation for chg and data

### PR DESCRIPTION
**Pull request by iamigo**
Add tests for RealtimeChangeHandler arg validation

### CODE REVIEW

File: MultiTable/test/RealtimeChangeHandler.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    const chg = { 'sample.add': { name: 'a|b', aspect: { name: 'b'} } };
+    RealtimeChangeHandler.handle(chg, data);
+    expect(data instanceof SubjectGroups).to.equal.true;
+    expect(data).to.have.property('rootSubject', 'a');
+    expect(data.map).to.have.property('a');
+  });
+
+  describe('handle: validateData >', () => {
+    it('instance of wrong class', () => {
+      expect(() => RealtimeChangeHandler.handle({}, new SubjectGroup('x')))
```
Comment (jgraff2) [inline MultiTable/test/RealtimeChangeHandler.js]: is there a reason for using arrow functions here?

Comment (iamigo) [reply MultiTable/test/RealtimeChangeHandler.js]: I was just following the chai docs for expect(...).to.throw(...)

File: MultiTable/test/RealtimeChangeHandler.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    it('undefined', () => {
+      expect(() => RealtimeChangeHandler.handle(undefined, data))
+      .to.throw(/MultiTable|RealtimeChangeHandler.handle|Invalid arg "chg"/);
+    });
+
+    it('array', () => {
+      expect(() => RealtimeChangeHandler.handle([], data))
+      .to.throw(/MultiTable|RealtimeChangeHandler.handle|Invalid arg "chg"/);
+    });
+  });
```
Comment (jgraff2) [inline MultiTable/test/RealtimeChangeHandler.js]: More tests for when it has sample.update or subject.update but no .new.

File: MultiTable/lens.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "1.0.18": "Show aspect help links and related links in sample modal",
     "1.0.19": "Aspect name case insensitivity - missed one",
     "1.0.20": "Add google analytics tracking for search, onclick modals and links, and toggle checkbox",
-    "1.0.21": "Prevent reverse tabnabbing"
+    "1.0.21": "Prevent reverse tabnabbing",
+    "1.0.22": "Add RealtimeChangeHandler arg validation and tests"
```
Comment (jgraff2) [inline MultiTable/lens.json]: If everyone's going to be making lots of concurrent changes, maybe we should hold off on updating this until all the changes are in.

File: MultiTable/test/RealtimeChangeHandler.js (hunk 2 above)
Comment (jgraff2) [inline MultiTable/test/RealtimeChangeHandler.js]: This PR is just for the args validation, right? You're also going to test the actual change handler functions?

Comment (iamigo) [reply MultiTable/test/RealtimeChangeHandler.js]: updated

Comment (iamigo) [reply MultiTable/test/RealtimeChangeHandler.js]: This PR is *just* for arg validation - I figured small PRs would be easier ... would you rather I wait and do one big PR?

File: MultiTable/test/RealtimeChangeHandler.js (hunk 1 above)
Comment (jgraff2) [reply MultiTable/test/RealtimeChangeHandler.js]: ok, I read it. Makes sense.

File: MultiTable/lens.json (hunk 3 above)
Comment (iamigo) [reply MultiTable/lens.json]: I'm OK with that, as long as the "last one in" adds meaningful release notes to the lens.json.

File: MultiTable/test/RealtimeChangeHandler.js (hunk 2 above)
Comment (jgraff2) [reply MultiTable/test/RealtimeChangeHandler.js]: Makes sense, just clarifying.

**Pull request merged**